include/dnnl.hpp

```diff
@@ -37,6 +37,33 @@
 #endif
 /// @endcond
 
+#ifndef MKLDNN_ENABLE_EXCEPTIONS
+ #if __cpp_exceptions || __EXCEPTIONS || \
```

Comment:
Can you please document which macros correspond to which compilers? `__cpp_exceptions` seems to be the C++ standard macro, `__EXCEPTIONS` seems to be the GCC macro (also supported by Clang and Intel C++ Compiler), and `_CPPUNWIND` is the MSVC macro.

Reply:
done